### PR TITLE
Added cpeters1982 keymap folder in lets_split

### PR DESCRIPTION
**Pull request by Cpeters1982**
Just adding my WIP keymap to the Let's Split directory.

### CODE REVIEW

**Comment by drashna** [comment]
Looks like you have some extra files from LUFA here.

Not really sure how to fix this.  @jackhumbert suggestions?

File: lib/lufa/Projects/TempDataLogger/TempLogHostApp/obj/Debug/TempLoggerHostApp.csproj.CoreCompileInputs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
+7fd46cf31393e6eb18b0c587f10292ac098d31c6
```
Comment (drashna) [inline lib/lufa/Projects/TempDataLogger/TempLogHostApp/obj/Debug/TempLoggerHostApp.csproj.CoreCompileInputs.cache]: Can you remove this manually?

File: lib/lufa/Projects/LEDNotifier/LEDMixerApp/obj/Debug/LEDMixer.csproj.CoreCompileInputs.cache [hunk 2]
```diff
@@ -0,0 +1 @@
+ce72ae334beda364398b52b2e8517c7c841969a9
```
Comment (drashna) [inline lib/lufa/Projects/LEDNotifier/LEDMixerApp/obj/Debug/LEDMixer.csproj.CoreCompileInputs.cache]: Can you remove this manually?

File: lib/lufa/Projects/LEDNotifier/CPUUsageApp/obj/Debug/CPUMonitor.csproj.CoreCompileInputs.cache [hunk 3]
```diff
@@ -0,0 +1 @@
+12ab9ae5ba4399a751617752b85a5fb09862e931
```
Comment (drashna) [inline lib/lufa/Projects/LEDNotifier/CPUUsageApp/obj/Debug/CPUMonitor.csproj.CoreCompileInputs.cache]: Can you remove this manually?

**Comment by Cpeters1982** [comment]
Sorry this took so long, Cleared.

**Comment by drashna** [comment]
Fantastic! 

Also, this isn't necesarry, but we've added official predefined rgbcolors here: 
https://github.com/qmk/qmk_firmware/blob/master/quantum/rgblight_list.h

But you'd need to use `rgblight_sethsv_blue()` to use these.  This is optional, but I figured that you'd appreciate knowing.

**Pull request merged**